src/test/scala/io/iohk/ethereum/vm/MockWorldState.scala

```diff
@@ -12,7 +12,8 @@ object MockWorldState {
 case class MockWorldState(
   accounts: Map[Address, Account] = Map(),
   codeRepo: Map[Address, ByteString] = Map(),
-  storages: Map[Address, MockStorage] = Map()
+  storages: Map[Address, MockStorage] = Map(),
+  headers: Map[BigInt, ByteString] = Map()
```

Comment:
Yes, and it should be the responsibility of the opcode to return 0 in that case. The state shouldn't be concerned with that. And in fact if the state returns 0, then that kind of invalidates the test, because if you change the comparison by mistake, the will not catch it.